src/platform/packages/shared/serverless/settings/search_project/index.ts

```diff
@@ -25,4 +26,5 @@ export const SEARCH_PROJECT_SETTINGS = [
   OBSERVABILITY_AI_ASSISTANT_SIMULATED_FUNCTION_CALLING,
   AI_ANONYMIZATION_SETTINGS,
   AGENT_BUILDER_ENABLED_SETTING_ID,
+  WORKFLOWS_UI_SETTING_ID,
```

Comment:
we will need to remove this for project settings because when workflow plugin isn't enabled, the uiSetting isn't registered and will complain on startup.